packages/ember-metal/lib/namespace_search.js

```diff
@@ -169,6 +169,7 @@ function tryIsNamespace(lookup, prop) {
     );
   } catch (e) {
     // continue
+    return false;
```

Comment:
so `tryIsNamespace` returns only `boolean`

Reply:
Hmm, seems like the return inside the `try` block returns an object, not boolean.  Am I misunderstanding something?

Reply:
my bad sorry